Rocket.Chat/Managers/Launcher/BugTrackingCoordinator.swift

```diff
@@ -9,10 +9,28 @@
 import Foundation
 import Fabric
 import Crashlytics
+import Instabug
 
 struct BugTrackingCoordinator: LauncherProtocol {
     func prepareToLaunch(with options: [UIApplicationLaunchOptionsKey: Any]?) {
         launchFabric()
+        launchInstabug()
+    }
+
+    private func launchInstabug() {
+        guard
+            let path = Bundle.main.path(forResource: "Keys", ofType: "plist"),
+            let keys = NSDictionary(contentsOfFile: path),
+            let instabug = keys["Instabug"] as? String
+        else {
+            return
+        }
+
+        #if BETA || DEBUG
+        Instabug.start(withToken: instabug, invocationEvent: .floatingButton)
```

Comment:
There's no floating button on my iPhone 🤔  also couldn't open it by shaking. Does it work without the key?